notebooks/cmb_inflation/bm_inflation.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<p>the use of resample mean is wrong in all the the jupyter notebook. let's talk on monday.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lambdaclass/finance_playground/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>